pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -297,6 +292,7 @@ buildStdenv.mkDerivation ({
   ++ lib.optionals enableDebugSymbols [ "--disable-strip" "--disable-install-strip" ]
 
   ++ lib.optional enableOfficialBranding "--enable-official-branding"
+  ++ lib.optional (lib.versionAtLeast version "95") "--without-wasm-sandboxed-libraries"
```

Comment:
@matthewbauer Looks like we need wasilibc for this firefox release, but I fail to build it. Can you take a look?

```
❯ nix-build -A wasilibc
error: assertion ((stdenv).targetPlatform != (stdenv).hostPlatform) failed at /home/hexa/git/nixos/master/pkgs/top-level/all-packages.nix:11781:25
(use '--show-trace' to show detailed location information)
```

Reply:
I tried hacking on this today a bit but can't figure out what's happening. I can't even eval the pkg in order to update it.

Reply:
I tried updating it and building it using `llvmPackages_13.stdenv` but then  I got 

```
clang-13: error: invalid thread model 'single' in '-mthread-model single' for this target
clang-13: error: invalid thread model 'single' in '-mthread-model single' for this target
clang-13: error: invalid thread model 'single' in '-mthread-model single' for this target
make: *** [Makefile:440: startup_files] Error 1
```